<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DescrForm</class>
 <widget class="QWidget" name="DescrForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>168</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>389</width>
    <height>168</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>389</width>
    <height>168</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DescrForm</string>
  </property>
  <widget class="QPushButton" name="bSave">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>140</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="labRemain">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>4</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Осталось 0 символов</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
